torch/onnx/symbolic_helper.py

```diff
@@ -939,6 +939,16 @@ def quantize_helper(g, tensor, scale, zero_point):
     output = g.op("QuantizeLinear", tensor, scale, zero_point)
     return g.op("prim::TupleConstruct", output, scale, zero_point)
 
+# NOTE: Original `bias` is float in PyTorch. Quantization is applied in kernel.
```

Comment:
Nit: can you move to a doc string inside the function def?

And Suggestions for content of the comment, I think this might be clearer:

```
In PyTorch, bias is float and is quantized implicitly inside the quantized ATen op kernel.
In ONNX we need to make the quantization explicit because operators expect all of their inputs to be quantized.
```